scripts/evaluate_cslb_features.py

```diff
@@ -0,0 +1,72 @@
+from multiprocessing.pool import Pool
```

Comment:
This is not a script, this is an example. Move it to the examples folder. Change fetched embeddings to just two exemplary (small ones).

Reply:
In structure is very similar to rest of scripts. As a example will be jupyter notebook.

Reply:
Not really. There are two scripts that are tools, one is for evaluating concrete embedding on all fast running experiments, second is for running specific embedding. Let's move it to examples.